bin/setup.js

```diff
@@ -1,11 +1,11 @@
-const spaceImport = require("contentful-import");
```

Comment:
While I basically agree with single quote over double quote, please do not reformat these files within this PR.